tools_galaxyp.yaml

```diff
@@ -1036,3 +1036,47 @@ tools:
   - name: metanovo
     owner: galaxyp
     tool_panel_section_label: Proteomics
+    
+  - name: cpt_fasta_remove_id
+    owner: cpt
+    tool_panel_section_label: FASTA/FASTQ
+    
+  - name: cpt_get_orfs
+    owner: cpt
+    tool_panel_section_label: Annotation
+    
+  - name: cpt_convert_mga
+    owner: cpt
+    tool_panel_section_label: Convert Formats
```

Comment:
`galaxyp` is specifically proteomics, I don't think they should be in this file at all.